app/services/cookie_handler.rb

```diff
@@ -0,0 +1,42 @@
+class CookieHandler
+
+  AUTH_URL = 'http://authentication.andela.com/'
+  DATA_PATH = 'loggedin'
+  COOKIE_KEY = 'andela:session'
```

Comment:
agree to this point, especially line 3, ought to be configured per environment!  authentication-staging.andela.com